k8s/custom-governance/cli_values_template.yaml

```diff
@@ -1,12 +1,12 @@
 # # This is a YAML-formatted file.
 # # Declare variables to be passed into your templates.
-aditum:
+cg:
   image:
     # Container registry repo where the application image is hosted.
     # Typically you do not need to modify this.
-    repo: marketplace.gcr.io/aditum-marketplace-dev/custom-governance
+    repo: gcr.io/aditum-marketplace-dev/custom-governance
```

Comment:
Actually, can you explain why `marketplace` prefix is removed in the new version?

Reply:
With 'marketplace' prefix, I get imagePullBackOff error saying the image is not available.

Reply:
Have you tried new location: gcr.io/cloud-marketplace/custom-governance/custom-governance ?

Reply:
No. Let me check and i'll update if it works